game/src/js/objects/characters/FireMonster.js

```diff
@@ -1,8 +1,8 @@
 import Character from './Character';
 
 export default class FireMonster extends Character {
-  constructor(scene, x, y) {
-    super(scene, x, y, 'fire_monster');
+  constructor(scene, x, y,colSize,offsetX,offsetY) {
```

Comment:
Since the colliders will probably be the same each time an instance of a character is created, these values should not be passed in when the character subclasses (e.g. Priest, Knight, etc.) are constructed. Instead, just put the respective values here in the `super()` method. For example,

```javascript
constructor(scene, x, y) {
  super(scene, x, y, 70, 95, 65);
  ...
}
```